specification.markdown

```diff
@@ -409,12 +409,16 @@ If the path contains an EBMLPathAtomRecursive part, the EBML Element can occur w
 
 As an example, a `path` of `1*(\Segment\Info)` means the element Info is found inside the Segment elements at least once and with no maximum iteration. An element SeekHead with path `0*2(\Segment\SeekHead)` may not be found at all in its Segment parent, once or twice but no more than that.
 
+The path value MUST be unique within the EBML Schema. So an `@id` found in the tree of EBML Elements can only match one path.
```

Comment:
I don't understand the last sentence. Is the `tree of EBML Elements` referring to an `EBML Schema` or an `EBML Document`? Also if the `@id` does not need to be unique, then couldn't it match more than one path?

Also I'd suggest to reference 'path' as \`@path\`.

Reply:
> Also I'd suggest to reference 'path' as `@path`.

OK to be coherent with the text around. But we never introducted the notion of @<something> which is an XML/XPath thing, not an EBML Schema thing.

Reply:
The `tree of EBML Elements` refers to the data found in an EBML Document.

The same `id` can be found in many places. The `id` is only used to find the semantic meaning of the data found at a given position in the tree. For example you can find 0xAA at `\root\element\tree\here` and `\root\element\tree\leaf\there`. They could have totally different meanings. only the path guarantees they are uniquely interpreted.